gen3/neural-networks/basic-example/main.py

```diff
@@ -0,0 +1,30 @@
+import depthai as dai
+from depthai_nodes import ParsingNeuralNetwork
+import time
+from utils.arguments import initialize_argparser
+
+arg_parser, args = initialize_argparser()
+
+visualizer = dai.RemoteConnection()
+
+deviceInfo = dai.DeviceInfo(args.ip)
+device = dai.Device(deviceInfo)
+
+with dai.Pipeline(device) as pipeline:
+    print("Creating pipeline...")
+    cameraNode = pipeline.create(dai.node.Camera).build(dai.CameraBoardSocket.CAM_A)
+    
+    networkNode = pipeline.create(ParsingNeuralNetwork).build(
```

Comment:
Let's be consistand with the [docs](https://rvc4.docs.luxonis.com/cloud/hubai/features/model-usage/) and call this `nn_with_parser`. Also `cameraNode` -> `camera_node` and `deviceInfo` -> `device_info` (generally I think snake case is preffered over camel case in python scripts)